cg/meta/orders/schema.py

```diff
@@ -203,7 +203,7 @@ def validate(self, *value):
 }
 
 FASTQ_SAMPLE = {
-    # Orderform 1508:18
+    # Orderform 1508:?
```

Comment:
What stands this ? for ?

Reply:
it was following # Orderform 1508:18 before, now I have made changes and I haven't verified against any particular orderform.. thoughts on what to do?

Reply:
naa, it's fine!